pkg/updater/gcs.go

```diff
@@ -386,6 +388,13 @@ func convertResult(log logrus.FieldLogger, nameCfg nameConfig, id string, header
 		out.Column.Extra = append(out.Column.Extra, val)
 	}
 
+	emailAddresses, ok := result.finished.Finished.Metadata[EmailListKey].([]string)
```

Comment:
Maybe  we should split this.
If we can't find Metadata[EmailListKey] set an empty list.
If the value is not '[]string' we should throw warning.
WDYT?

Reply:
We should log an error if invalid email list is given? I didn't saw any handling of invalid input (or handling errors) in the code.

Reply:
I think logging an error is fine here, yeah. (If it doesn't cast correctly to a []string then we should note that somehow).